### PR TITLE
Add phone number mask

### PR DESCRIPTION
**Pull request by jjeanjacques10**
Adding a mask to the phone number and avoiding letters in the input.

image

### CODE REVIEW

File: src/number-form/index.html [hunk 1]
```diff
@@ -49,7 +49,7 @@ <h1 id="title">WhatsApp Number</h1>
         </select>
         <input
           id="phone-number"
-          type=" number"
+          type="text"
```
Comment (Singh212dab) [inline src/number-form/index.html:52]: type="value"

Comment (Singh212dab) [reply src/number-form/index.html:52]: i think value will be good

**Review by Singh212dab — CHANGES_REQUESTED** [review]

File: src/number-form/index.html (hunk 1 above)
Comment (jjeanjacques10) [reply src/number-form/index.html:52]: Sorry, I didn't get it... is value a valid term to use in type attribute? In this case I'm using text to apply the mask into phone number.

**Comment by Singh212dab** [comment]
Yeah that will work too,You won't get error in any

**Comment by jjeanjacques10** [comment]
But why is good to use "type=value"? @Singh212dab

**Pull request merged**